index.d.ts

```diff
@@ -667,6 +668,58 @@ declare module '@amaas/amaas-core-sdk-js' {
     }[]
   }
 
+  export interface IEOD {
+    assetManagerId: string
+    assetId: string
+    businessDate: string
+    price: string
+    active: number
+    internalId: string
+    createdId: string
+    updatedTime: string
+    clientId: string
+    createdBy: string
+    updatedBy: string
+  }
+
+  export interface ICurve {
+    assetManagerId: string  
+    assetId: string
+    curveType: string
+    fixingType: string
+    businessDate: string
+    curveTimestamp: string
+    curveRates: {}
+    additional: {}
```

Comment:
You are declaring `curveRates` and `additional` as literal empty objects, meaning they can't be anything except empty objects.